cam

```diff
@@ -0,0 +1,279 @@
+src/EventManager.cpp:732:	// When an event is delayed, we create a copy and push it to mapr->delayed_events.
```

Comment:
I'm guessing this file was accidentally committed?

Reply:
That's right, i forgot to remove it.

Reply:
That's right, i forgot to remove it. It shouldn't be there.